src/shared/components/case-editor/domain/wizard-page-field-complex-mask.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Orderable } from '../../../domain/order/orderable.model';
+
+export class ComplexFieldMask implements Orderable {
```

Comment:
perhaps `ComplexFieldOverride` or even `ComplexSubfieldOverride` helps better understanding what this is doing?